webapp/src/Controller/API/ContestController.php

```diff
@@ -428,15 +428,15 @@ public function getContestYamlAction(Request $request, string $cid): StreamedRes
         $penalty_time = $this->config->get('penalty_time');
         $response     = new StreamedResponse();
         $response->setCallback(function () use ($contest, $penalty_time) {
-            echo "name:                     " . $contest->getName() . "\n";
```

Comment:
Should we use the getCOntestYamlData from the import export service here for less duplication?

Reply:
We should yes, it will also remove this annoying whitespace issue.